File: JavaScript/read-lists.js
import { readFileSync } from 'fs';

try {
    const raw = readFileSync('./list.json', 'utf8');
    const data = JSON.parse(raw);

    if (Array.isArray(data.lists)) {
        data.lists.forEach(({ id, name }) => {
            console.log(`ID: ${id}, Name: ${name}`);
        });
    } else {
        console.error('Поле "lists" відсутнє або не є масивом.');
    }
} catch (error) {
    console.error('Помилка:', error.message);
}

File: PythonProject/read_lists.py
import json

# Відкриваємо та читаємо файл lists.json
with open('lists.json', 'r', encoding='utf-8') as file:
    data = json.load(file)

# Отримуємо масив зі словника
lists = data.get("lists", [])

# Виводимо id та name кожного елементу
print("➡️ ID та Name кожного списку:")
for item in lists:
    print(f"ID: {item['id']}, Name: {item['name']}")